packages/material-ui/src/Link/Link.js

```diff
@@ -139,15 +139,7 @@ Link.propTypes = {
    * The color of the link.
    * @default 'primary'
    */
-  color: PropTypes.oneOf([
-    'error',
-    'inherit',
-    'initial',
-    'primary',
-    'secondary',
-    'textPrimary',
-    'textSecondary',
-  ]),
+  color: PropTypes /* @typescript-to-proptypes-ignore */.any,
```

Comment:
Needed for fixing https://app.circleci.com/pipelines/github/mui-org/material-ui/36615/workflows/0ea8cae7-7ee2-4350-881b-06df6d568b69/jobs/217632